bin/exec.ml

```diff
@@ -66,8 +66,8 @@ let term =
       |> List.map ~f:(fun p -> Target.Path p)
       |> Target.resolve_targets_mixed common setup
       |> List.concat_map ~f:(function
-        | Ok targets -> targets
-          | Error _ -> []) )
```

Comment:
Misaligned

Reply:
This is how `$ make fmt` chooses to do it :/ Let me make sure I have the most recent ocamlformat